config/api.yml

```diff
@@ -1659,11 +1659,13 @@
     :identifier: network_router
     :options:
     - :collection
+    - :subcollection
```

Comment:
IIRC, this means that `NetworkRouters` can be a subcollection of something else.